mlir/tools/mlir-miopen-lib/mlir-miopen-lib.cpp

```diff
@@ -55,6 +55,28 @@ static void strToTokens(const std::string &arguments,
     }
   }
 }
+
```

Comment:
```C++
#if __cplusplus < 201103L
#error ...
#endif
```

Reply:
I can add your link above as comments but macro is probably not necessary because compiling LLVM requires C++11 since 7 years ago, [here](https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/blame/miopen-dialect/llvm/docs/CMake.rst#L762-L765).

Reply:
Okay.

Reply:
Resolved